lua.spec

```diff
@@ -2,16 +2,23 @@
 
 Name:       lua
 Summary:    Powerful light-weight programming language
-Version:    5.3.5
+Version:    5.4.4
 Release:    4
 License:    MIT
-URL:        https://www.lua.org/
+URL:        https://github.com/sailfishos/lua
 Source0:    https://www.lua.org/ftp/lua-%{version}.tar.gz
```

Comment:
We should really prefer here %{name}-%{version}.tar.xz as the url is anyway not right because of how we build the tarballs.

Reply:
Sure.